module/Category/src/Controller/CategoryControllerFactory.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
```

Comment:
Fine for a training task, but please be careful not to commit lines like this in a real project.